packages/design-system/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -7,6 +7,15 @@ import classNames from 'classnames';
 export class ChoiceList extends React.PureComponent {
   constructor(props) {
     super(props);
+
+    if (process.env.NODE_ENV !== 'production') {
+      if (props.multiple) {
+        console.warn(
+          `[Deprecated]: Please remove the 'multiple' prop in <ChoiceList>, use 'type' instead. This prop is obsolete and will be removed in a future release.`
```

Comment:
could you update this to say 
```
use `type="checkbox"` instead
``` 
Also please replace "obsolete" with "deprecated"

Reply:
Noted, have changed the wording to deprecated.